crates/shared/src/sources/balancer_v2/pools/stable.rs

```diff
@@ -108,6 +111,48 @@ impl FactoryIndexing for BalancerV2StablePoolFactory {
     }
 }
 
+// TODO - this probably doesn't need to be so clearly duplicated.
```

Comment:
Ben and I talked about this. I felt this was fine for now. When there is a V3 it might be worth it to deduplicate.